src/main/java/frc/robot/subsystems/ShooterSystem.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.*;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class ShooterSystem extends SubsystemBase {
+    private final CANSparkMax motorLT;
+    private final CANSparkMax motorRT;
+    private final CANSparkMax motorLB;
+    private final CANSparkMax motorRB;
+    private final RelativeEncoder encoderLT;
+    private final RelativeEncoder encoderRT;
+    private final RelativeEncoder encoderLB;
+    private final RelativeEncoder encoderRB;
+    private final SparkPIDController pid;
+    private final double targetRPM;
+
+    public ShooterSystem(double targetRPM) {
```

Comment:
you should not get the targetRPM as a system parameter ,it should be a parameter in the method that uses pid and the command that should rotate the shooter by the pid .
that is because we should be able to have different tragetRPM in different situations.

```suggestion
    public ShooterSystem() {
```

Reply:
done